### DISCUSSION TITLE
Serilog Sink

### DISCUSSION THREAD

**Discussion by Depechie** [open-telemetry/opentelemetry-dotnet#3857] [General]
Is there any roadmap timeline for the logging branch being merged into main?
Because we would love to try out the [https://github.com/open-telemetry/opentelemetry-dotnet/blob/main-logs/src/OpenTelemetry.Extensions.Serilog/OpenTelemetrySerilogSink.cs](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main-logs/src/OpenTelemetry.Extensions.Serilog/OpenTelemetrySerilogSink.cs).

@CodeBlanch seems you are most active on the branch?

**Comment by CodeBlanch** [comment]
Relates to #3602 

@Depechie Not really, no. The problem is the [spec is currently marked as experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/08a15efd4c43336ae31d14809aaa1acce07c9a0a/specification/logs/api.md). We can't release anything until we get to stable or close 😢

We are working with the spec to try and get there, just no timeline yet.

/cc @alanwest @jack-berg

**Reply by Depechie** [reply]
Thx for the details @CodeBlanch !
Of course not the answer I was hoping for, but now you know devs are really eager on trying out that feature ;)
